plugin/evm/vm.go

```diff
@@ -905,14 +908,30 @@ func attachEthService(handler *rpc.Server, apis []rpc.API, names []string) error
 	return nil
 }
 
+var orderBookContractFileLocation = "contract-examples/artifacts/contracts/hubble-v2/OrderBook.sol/OrderBook.json"
+var orderBookContractAddress = common.HexToAddress("0x0300000000000000000000000000000000000069")
+
+func SetOrderBookContractFileLocation(location string) {
+	orderBookContractFileLocation = location
+}
+
 func (vm *VM) NewLimitOrderProcesser() LimitOrderProcesser {
+	memoryDb := limitorders.NewInMemoryDatabase()
+	jsonBytes, _ := ioutil.ReadFile(orderBookContractFileLocation)
+	orderBookAbi, err := abi.FromSolidityJson(string(jsonBytes))
```

Comment:
There's a way to generate go bindings for a contract abi.

Reply:
@shubhamgoyal42  experimented with a bit. We will pick it up separately